.ado/variables/shared.yml

```diff
@@ -1,8 +1,18 @@
 variables:
+  # Injection configuration
   runCodesignValidationInjection: false
+  
+  # Enables `chalk` to show colored output to Azure Pipelines
+  FORCE_COLOR: 3
```

Comment:
>3 [](http://example.com/codeflow?start=15&length=1)

I just wonder what this magic number 3 means.

Reply:
Documented (sort of) here: https://github.com/chalk/chalk#supportscolor

I think the difference between 1, 2, 3 is level of different colors supported. Bits seem to work correctly when exposing all colors.

Reply:
Oh.. there's a table there that I missed.

| Level | Description |
| :---: | :--- |
| `0` | All colors disabled |
| `1` | Basic color support (16 colors) |
| `2` | 256 color support |
| `3` | Truecolor support (16 million colors) |

Reply:
Is there a reason why this coloring stuff isn't it's own PR?

Reply:
This PR was going to pass `--color` to more scripts. I realized I could instead make it better for other scripts by changing the more global config.

Reply:
Basically, wanted to avoid less of this hackage:
![image](https://user-images.githubusercontent.com/835219/141259797-b6ab0273-0eb8-486b-b52c-cab6eb2f98a4.png)